src/guide/plugins.md

```diff
@@ -12,49 +12,77 @@ Plugins usually add global-level functionality to Vue. There is no strictly defi
 
 5. A library that provides an API of its own, while at the same time injecting some combination of the above (e.g. [vue-router](https://github.com/vuejs/vue-router)).
 
-## Using a Plugin
+## Writing a Plugin
+
+In order to better understand how to create your own Vue.js plugins, we will create a very simplified version of a plugin that displays `i18n` ready strings. 
 
-After a Vue app has been initialized with `createApp()`, you can add a plugin to your application by calling the `use()` method:
+A Vue.js plugin is either an `object` that should expose an `install()` method, or a `function`. 
```

Comment:
```suggestion
```

I could recommend moving this to the top where we define what a Plugin is.

```
A Vue.js plugin usually adds global-level functionality to Vue. It is either an `object` that should expose an `install()` method, or a `function`. There is no strictly defined scope for a plugin, but common scenarios where plugins are useful include:

Reply:
Agreed, going to add a bit more to the description though